bootstrap/src/main/config/mediaserver.xml

```diff
@@ -21,6 +21,7 @@
 	<controller protocol="mgcp">
 		<address>127.0.0.1</address>
 		<port>2427</port>
+		<mgcpBuffer size="5000" />
```

Comment:
use `<bufferSize>5000</bufferSize>` instead

Reply:
Is "bufferSize" not a general name?  What about some kind of specific name which is easy to refer in documentation.

Reply:
I guess you have a point, we may call it `channelBuffer` for example